students/kosarzewski_maciej/lesson_01_basics/apple_sharing.py

```diff
@@ -0,0 +1,6 @@
+# Read the numbers like this:
+students = int(input())
```

Comment:
It will be great to include text description whenever user is prompt for input
students = int(input("Provide numer of students: "))